docs/howto/add-identify.md

```diff
@@ -1,12 +1,114 @@
 # How-to add Identify
 
-Identify is supported on `wms`, `mapserver`, `wfs`, `mapserver-wfs`, and `ags-vector` map-source types.
+If you would like to have your map users to be able to get information on specific features using the Identify tool,
+you must:
+* ensure the `<map-source>` type is one of: `wms`, `mapserver`, `wfs`, `mapserver-wfs`, or `ags-vector`
```

Comment:
Shouldn't identify be supported on `geojson` too?

Reply:
Yes, but for some reason it wasn't listed in the doc.  That led me to today's debug session.  It should be listed in the doc, and the bug #522  needs to be fixed to make it work.